### PR TITLE
Stub in Public Key Cryptography level type

### PR DESCRIPTION
**Pull request by islemaster**
The **Public Key Cryptography Widget** is a new, simple widget level (much like the Odometer and Text Compression widgets) that will teach students about the mathematics behind public key cryptography.  [Spec here (requires login)](https://docs.google.com/document/d/1EIu6BFB2Q-AgZC3fVJ7Dr1BWRGdEfjdVhO0aoBPoFyo/edit#).  Here's an early concept from the spec:

image

This PR don't implement any of the UI or functionality for the widget though - it just does all the boilerplate prep work.
- Create a new level type for the widget
- Make it create-able via levelbuidler
- Have its JavaScript and stylesheet generated via our apps build
- Create a new stage & level in the AllTheThings script
- Add a basic eyes test over the widget initial appearance

So all this work leads to the creation of this level:

screenshot from 2016-08-10 13-56-34

Which is not very exciting (yet) but is just the blank slate I need :grin:.

### CODE REVIEW

File: apps/Gruntfile.js [hunk 1]
```diff
@@ -438,6 +439,14 @@ module.exports = function (grunt) {
...
         pd: './src/code-studio/pd/workshop_dashboard/workshop_dashboard.jsx'
       }
+    },
+
+    {
+      uniqueName: 'publicKeyCryptography',
+      entries: {
+        publicKeyCryptography: './src/publicKeyCryptography/publicKeyCryptography.js'
+      },
+      provides: ['react', 'react-dom', 'radium']
```
Comment (Bjvanminnen) [inline apps/Gruntfile.js:449]: It doesn't look like publicKeyCryptography does provide these?

Comment (islemaster) [inline apps/Gruntfile.js:449]: Oh, it doesn't yet (whoops) but I do expect it to soon - I want to use React to write the widget, and the `publicKeyCryptography` bundle will stand on its own, not requiring the apps common file to work.  Modeled on [the embedBlocks bundle, here](https://github.com/code-dot-org/code-dot-org/pull/9964/files#diff-26cdc4306e5d33ea4df1299321d8852aR422).

Having this here for now doesn't seem to cause any problems - I believe it just prevents aliasing those requires if I were to use them.

Comment (Bjvanminnen) [inline apps/Gruntfile.js:449]: Yeah, should be fine to leave them. Wanted to make sure you didn't misunderstand how this worked and thought that by adding this line you would suddenly get these provided to you. Sounds like your understanding is on point :)

**Comment by Bjvanminnen** [comment]
lgtm from my perspective

**Comment by joshlory** [comment]
LGTM.  Good idea to symlink the `/dashboard/public` files so we can use the apps build!

**Pull request merged**